shared/VCMetadata.ts

```diff
@@ -13,6 +13,7 @@ export class VCMetadata {
   issuer?: string = '';
   protocol?: string = '';
   static vcKeyRegExp = new RegExp(VC_ITEM_STORE_KEY_REGEX);
+  downloadCounter?: number = 0;
```

Comment:
can we remove downloadCounter from vcmetadata ?

Reply:
moved to context